eu.fortysixandtwo.chessclock.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id" : "eu.fortysixandtwo.chessclock",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "chess-clock",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+	{
+            "name": "gsound",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/gsound.git"
+                }
+            ],
+            "modules": [
+                {
+                    "name": "libcanberra",
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "http://0pointer.de/lennart/projects/libcanberra/libcanberra-0.30.tar.xz",
+                            "sha256": "c2b671e67e0c288a69fc33dc1b6f1b534d07882c2aceed37004bf48c601afa72"
+                        }
+                    ],
+                    "config-opts": [
+                        "--disable-alsa",
+                        "--disable-null",
+                        "--disable-oss"
+                    ]
+                }
+            ]
+        },
+        {
+            "name" : "chess-clock",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/World/chess-clock.git",
+		    "tag" : "v0.6.1"
```

Comment:
please indent properly. you are not the only one that need to read this.

Reply:
Telling my editor to reindent everything this is what I get.

Please point out what specifically is wrong or how it should look like.

Reply:
If the indentatio here being incorrect is not obvious I'm not sure what else to say:

![image](https://github.com/user-attachments/assets/524ac80c-cad7-4e10-a380-50633d4e981a)

Fortunately unlike yaml or python, indentation isn't semantic. But this make things unreadable.

Reply:
As mentioned above, all the tabs need to be replaced with spaces. It's fully possible that this looks correctly indented to @fortysixandtwo if they use a tab width of 8. If the tab characters are converted to 8 spaces, it'll look OK for everyone.